CS342MemberViews/Package.swift

```diff
@@ -19,6 +19,7 @@ let package = Package(
     products: [
         .library(name: "PaulSchmiedmayer", targets: ["PaulSchmiedmayer"]),
         .library(name: "OliverAalami", targets: ["OliverAalami"]),
+        .library(name: "AbhinavAgarwal", targets: ["AbhinavAgarwal"])
```

Comment:
Please add a comma after your `library` statement to fix the build error.
```suggestion
        .library(name: "AbhinavAgarwal", targets: ["AbhinavAgarwal"]),
```

Reply:
Hi, Paul! This was the old request. I would like a review of request #42 which passes all the tests. As for the creativity part, I added animated apple watch activity rings around the profile picture since I'm doing the PAWS project.

Reply:
@abhinav30219 Thank you for pointing me to the updated PR, I will take a look at that 👍